packages/core/src/payment-integration/default-payment-integration-service.ts

```diff
@@ -154,4 +158,12 @@ export default class DefaultPaymentIntegrationService implements PaymentIntegrat
 
         return this._storeProjection.getState();
     }
+
+    async createBuyNowCart(buyNowCartRequestBody: BuyNowCartRequestBody): Promise<Cart> {
```

Comment:
https://github.com/bigcommerce/checkout-sdk-js/pull/1791/files#diff-a9aaa3275847637b88e61e83799c3ca27bb47afe157ff50fe27345ea9f6e4042R58 we have options defined in the interface but not here? Is that intentional?

Reply:
I forgot to add options for this method. Yeah, the `options?: RequestOptions` should be added here

Reply:
Fixed